parse/parse.go

```diff
@@ -23,6 +23,16 @@ func (p *parser) next() item {
 	return p.items[p.pos]
 }
 
+func (p *parser) peekNonWS() item {
```

Comment:
peekNonWs? Could you add a description?

Reply:
peek next token which is not whitespace, should be pretty self explanatory, but i will add godoc